Sources/Tokenizer.swift

```diff
@@ -10,6 +10,18 @@ extension String {
     var singleQuoteCount = 0
     var doubleQuoteCount = 0
 
+    let specialCharacters = ",|:"
+    func appendWord(_ word: String) {
+      if components.count > 0 &&
+        (specialCharacters.characters.contains(components.last!.characters.last!) ||
```

Comment:
😱 all those force unwraps 😢

Reply:
🤷‍♂️ it's all safe 🙄

Reply:
`if let lastCompChar = components.last?.characters.last, (specialCharacters.contains(lastCompChar) || …)`

Reply:
@AliSoftware done